Sources/Remote Logging/Crash Logging/CrashLogging.swift

```diff
@@ -53,6 +53,21 @@ public class CrashLogging {
 
             /// Attach stack traces to non-fatal errors
             options.attachStacktrace = true
+
+            // Performance monitoring options
+            options.enableAutoPerformanceTracking = self.dataProvider.enableAutoPerformanceTracking
+            options.tracesSampler = { _ in
+                // To keep our implementation as Sentry agnostic as possible, we don't pass the
+                // input `SamplingContext` down the chain.
+                NSNumber(value: self.dataProvider.tracesSampler())
+            }
```

Comment:
I _could_ move all this closure wrapping logic into the `dataProvider.tracesSampler()` definition. But I think it fit better here where all the code is Sentry-specific.